app/app.config.ts

```diff
@@ -8,37 +8,52 @@ import { version } from './package.json'
 module.exports = {
   expo: {
     name: 'STA',
-    displayName: 'STA Volunteers',
-    slug: 'sta',
+    slug: 'volunteer-app',
     version,
     orientation: 'portrait',
     icon: './assets/sta-icon.png',
-    userInterfaceStyle: 'light',
+    userInterfaceStyle: 'automatic',
     splash: {
       image: './assets/splash-screen.png',
       resizeMode: 'contain',
       backgroundColor: '#ffffff',
     },
     assetBundlePatterns: ['**/*'],
-    ios: {
-      supportsTablet: true,
-    },
     android: {
       adaptiveIcon: {
         foregroundImage: './assets/sta-icon.png',
         backgroundColor: '#ffffff',
       },
+      package: 'org.scottishtecharmy.volunteerapp',
+      versionCode: 14,
+    },
+    ios: {
+      bundleIdentifier: 'org.scottishtecharmy.volunteerapp',
+      infoPlist: {
+        ITSAppUsesNonExemptEncryption: false,
+        NSCalendarsUsageDescription:
+          "The app will show tech/volunteering-related events that the user might be interested in. The user can choose to attend an event - if so, the app needs access to the user's calendar in order to add an event.",
```

Comment:
Does this bit need to be single quotes or nah?

Reply:
Good spot, thanks @vglampard -- yep almost all strings should use single quotes, the exception like here (and this is enforced by Prettier/ES Lint in our code editors) is when there's an apostrophe (a.k.a. single quote) in the string, in which case we use double quotes.